projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -148,11 +148,7 @@ public static function endpoint_deprecated_warning( $function_name, $deprecated_
 
 		$messages[] = esc_html__( 'Please update all the Jetpack plugins to the latest version.', 'jetpack-publicize-pkg' );
 
-		_doing_it_wrong(
-			esc_html( $function_name ),
-			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- We have done it above.
-			implode( ' ', $messages ),
-			'$$next-version$$'
-		);
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- We have done it above.
+		_doing_it_wrong( esc_html( $function_name ), implode( ' ', $messages ), 'package-$$next-version$$' );
```

Comment:
By hard-coding it here, if you deprecate a different endpoint in a future version you can't use this function. Perhaps the version should be a param for the function?

Reply:
Also, if some plugin calls this endpoint importing this package, the version number would look something like `package-3.2.1`, which isn't too helpful when troubleshooting. :^)

Reply:
Yeah, I think it's better to use the plugin prefix instead.